src/types/base.ts

```diff
@@ -60,7 +60,7 @@ export interface Metric {
    * The array may also be empty if the metric value was not based on any
    * entries (e.g. a CLS value of 0 given no layout shifts).
    */
-  entries: (PerformanceEntry | LayoutShift)[];
+  entries: PerformanceEntry[] | LayoutShift[];
```

Comment:
```suggestion
  entries: PerformanceEntry[];
```

Now that all of the other metric interfaces set their own `entires` type, I think this can be simplified.